src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -402,6 +404,11 @@ public string ResolveVersion(string versionToResolve)
         /// <returns>Package json Object.</returns>
         internal static dynamic GetPackageJsonObject(ISourceRepo sourceRepo, ILogger logger)
         {
+            if (!sourceRepo.FileExists(NodeConstants.PackageJsonFileName))
```

Comment:
Noticed quite a few filenotfound exception for this file, so adding this check to avoid that.